README.md

```diff
@@ -0,0 +1,82 @@
+# Senior Game
+### Senior Design CEN4914 Spring 2021
+
+# Dependencies
+
+- Git LFS
+- Rustup
+- `lld` (Install via package manager on linux, use [cargo-binutils](https://github.com/rust-embedded/cargo-binutils) if on Windows)
+
+# Getting Started
+
+## Setting up Rust
+
+```
+# Install Rust Toolchain via Rustup
+
+rustup toolchain install stable
+
+# If on Linux, install cargo-binutils
+
+cargo install -f cargo-binutils
+
+# Install LLD Linker Component
+
+rustup component add llvm-tools-preview
+
+```
+
+## Using rustfmt
+
+```
+# Install nightly Rust toolchain
+
+rustup toolchain install nightly
```

Comment:
Why do we install nightly and stable?

Reply:
I'm unsure of the rust echo system in general but I was wary of running the game on the nightly toolchain. However from looking now [it appears they test against the nightly toolchain so we should be ok to only do nightly](https://github.com/bevyengine/bevy/actions/runs/334886604/workflow).

Reply:
We want nightly in this case, it's measurably faster than stable